docs/content/en/docs/fleet-manager/create-attachedcluster.md

```diff
@@ -0,0 +1,127 @@
+---
+title: "Integrating the AttachedClusters with Fleet"
+linkTitle: "Integrating the AttachedClusters with Fleet"
+weight: 15
+description: >
+  Your roadmap to integrating and managing AttachedClusters with Kurator Fleet.
+---
+
+In Kurator, clusters that were not created by Kurator are referred to as `AttachedClusters`.
+These clusters can be managed by adding them to the Kurator Fleet, expanding the fleet's control to clusters not originally created by Kurator.
+
+This guide will walk you through the process of creating AttachedCluster resources, using two Kind clusters as examples.
+
+## Prerequisites
+
+### Cluster Operator and kind Cluster
+
+As the AttachedCluster object is controlled by the cluster-operator, you need to first go to [install cluser operator](/docs/setup/install-cluster-operator.md) page to create kind clusters using `hack/local-dev-setup.sh` and install cluster operator.
```

Comment:
> the AttachedCluster object is controlled by the cluster-operator ?

Reply:
yes, currentlly the AttachedCluster object is controlled by the cluster-operator .
although AttachedClusters are primarily used in the context of Fleet, they should be considered equivalent to regular Clusters and managed in the same way by the Cluster Operator